core/src/replay_stage.rs

```diff
@@ -474,6 +474,32 @@ impl ReplayStage {
                     total_epoch_stakes,
                     stake_lockout.lockout(),
                 );
+                let percentage_stake: f64 =
+                    stake_lockout.stake() as f64 / total_epoch_stakes as f64;
+                let fork_metrics = *fork as i64;
+                solana_metrics::submit(
```

Comment:
As discussed, I think it would be good to report the parent slot too and show that in the table